server/repositories/cmsQueries/audioPageQuery.js

```diff
@@ -76,7 +76,7 @@ class AudioPageQuery {
       seriesName: item.fieldMojSeries?.name,
       seriesSortValue: item.seriesSortValue,
       media: item.fieldMojAudio?.uri?.url,
-      categories: getCategoryIds(item.fieldMojTopLevelCategories),
+      categories: getCategoryId(item.fieldMojTopLevelCategories),
```

Comment:
guess it would be nice to update this `categories` to `category` but obviously not for this PR

Reply:
So I started, but it snowballed and then it gets more confusing, because this is the title used in the analytics. and changing that will mess with that. We can change it everywhere else and just not in the analytics title of course?

Reply:
Seems sensible, chip away at the refactoring?